x-pack/plugins/maps/common/descriptor_types/map_descriptor.ts

```diff
@@ -7,10 +7,11 @@
 
 /* eslint-disable @typescript-eslint/consistent-type-definitions */
 
+import type * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
```

Comment:
Is there a reason we can't import the `GeoShapeRelation` type directly?

```suggestion
import type { GeoShapeRelation } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
```